### DISCUSSION TITLE
The time zone of the datetime of the Open API response

### DISCUSSION THREAD

**Discussion by YueXiaolong** [etsy/open-api#1175] [Q&A]
The date and time of the Open API response are all timestamps. The specific time zone information cannot be confirmed, and there is no task description in the document.
![image](https://github.com/etsy/open-api/assets/8215105/1dea9e5c-ddc9-486b-a9b1-0062297f009b)

**Comment by mjec** [comment]
In case anyone comes across this thread, all timestamps are number of seconds since the Unix Epoch (midnight UTC on January 1, 1970). They do not have an associated time zone. See https://en.wikipedia.org/wiki/Unix_time for more on this format.
